docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,85 @@
+</DocCallOut>
+10. Click **Analyze logs**, then wait for processing to complete. This may take several minutes.
+11. After processing is complete, the pipeline's field mappings appear, including both ECS and custom fields.
+
+<DocImage url="images/auto-import-review-integration-page.png" alt="The Automatic Import Review page showing proposed field mappings" />
+
+12. (Optional) After reviewing the proposed pipeline, you can finetune it by clicking **Edit pipeline**. Refer to the [((elastic-sec)) ECS reference](https://www.elastic.co/guide/en/security/current/siem-field-reference.html) to learn more about how field mappings should look. When you're satisfied with your changes, click **Save**. 
+
+<DocImage url="images/auto-import-edit-pipeline.gif" alt="A gif showing the user clicking the edit pipeline button and viewing the ingest pipeline flyout" />
+
+<DocCallOut title="Tip">
+You can use the <DocLink slug="/serverless/security/data-quality-dash" text="Data Quality dashboard"/> to check the health of your data ingest pipelines and field mappings.
+</DocCallOut>
+
+13. Click **Add to Elastic**. After the **Success** message appears, your new integration will be available on the Integrations page. 
+
+<DocImage url="images/auto-import-success-message.png" alt="The Automatic Import success message" />
+
+14. Click **Add to an agent** to deploy your new integration and start collecting data, or click **View integration** to view detailed information about your new integration. 
+
+<DocCallOut title="Note">
+Once you've added an integration, you can't edit any details other than the ingest pipeline, which you can edit by going to **Stack management → Ingest pipelines**. 
+</DocCallOut>
+
+<DocCallOut color="warning" title="Important">
+Using Automatic Import allows our customers to create new third-party data integrataions through the use of third-party generative AI models (“GAI models”). Any third-party GAI models that you may choose to use are owned and operated by their respective providers. Elastic does not own or control these third-party GAI models, nor does it influence their design, training or data handling practices. Using third party GAI models with Elastic solutions, and using your data with third-party GAI models is at your discretion. Elastic bears no responsibility or liability for the content, operation, or use of these third-party GAI models, nor for any potential loss or damage arising from their use. Users are advised to exercise caution when using GAI models with personal, sensitive, or confidential information, as data submitted may be used for training of the models or other purposes. Elastic recommends familiarizing yourself with the development practices and terms of use of any third-party GAI models before use.
+
+You are responsible for ensuring that your use of Automatic Import complies with the terms and conditions of any third-party platform you connect with.
+</DocCallOut>
```

Comment:
Seems like this big honkin warning should be at the top of the page, _before_ the user goes all-in and hands over their data by going through the steps above. It feels really buried down here, almost like we don't want folks to see it, even though we actually want the opposite.

Reply:
Yes, agree 💯